src/transformers/pipelines/base.py

```diff
@@ -630,7 +639,14 @@ def check_model_type(self, supported_models: Union[List[str], dict]):
                 The list of models supported by the pipeline, or a dictionary with model class values.
         """
         if not isinstance(supported_models, list):  # Create from a model mapping
-            supported_models = [item[1].__name__ for item in supported_models.items()]
+            supported_models_names = []
+            for config, model in supported_models.items():
+                # Mapping can now contain tuples of models for the same configuration.
+                if isinstance(model, tuple):
+                    supported_models_names.extend([_model.__name__ for _model in model])
+                else:
+                    supported_models_names.append(model.__name__)
+            supported_models = supported_models_names
```

Comment:
This was forgotten with the addition of the possibility to have two different models for a single configuration in https://github.com/huggingface/transformers/pull/11150.

Reply:
Good catch!